rust/headless-client/src/ipc_service.rs

```diff
@@ -414,6 +415,13 @@ impl<'a> Handler<'a> {
                 .as_mut()
                 .context("No connlib session")?
                 .set_dns(v),
+            ClientMsg::SetDisabledResources(disabled_resources) => {
+                self.connlib
+                    .as_mut()
+                    .context("No connlib session")?
+                    .set_disabled_resources(disabled_resources);
+                self.dns_controller.flush()?;
```

Comment:
I don't think we need to do this here. `set_disabled_resources` is async because we are going through a channel. Are we not calling `flush` whenever we receive updated resources? IMO that would make more sense.

Reply:
disabling resources doesn't call update resources only update routes. But I'm in favor of doing it there, it makes more sense.